app/js/thisissoon-core/app.js

```diff
@@ -36,9 +37,11 @@ angular.module("thisissoon.core", [
      * @param {Service} $rootScope
      * @param {Service} CacheService
      */
-    function ($rootScope, CacheService, snSkrollr, ENV) {
+    function ($rootScope, $window, CacheService, snSkrollr, ENV) {
 
-        snSkrollr.init();
+        if ($window.innerWidth > 1023) {
+            snSkrollr.init();
+        }
 
```

Comment:
I think it has `skrollr.destory()` so linking with the ResizeService could work.